simple-request-response/README.md

```diff
@@ -0,0 +1,75 @@
+# A simple request/response abstraction
```

Comment:
I'd suggest to drop the "simple" keyword.

Request-response implies that only one message will be sent in each direction. Consequently, I think it is save to require closing the write-side after each message (see other comment below). Thus, I believe this is the most-basic form of request-response and I don't see there being another primitive.

To conclude: I don't think there will be another req-res primitive that isn't compatible with this one, meaning we don't need a qualifier but can just say that this is how you do any form of request-response on top of libp2p streams.

Reply:
I see "request-response" being a more general idea. And this is a specific implementation of it. We could imagine alternative implementations that have associated metadata (like HTTP headers), or specific encodings (like an RPC system), or allow pipelining.

I want to be able to talk about "request-response" protocols without also talking about a specific implementation.

I'm not attached to the "simple" here. But I'd prefer to not use "request-response" directly.

Reply:
Based on [this](https://github.com/libp2p/specs/pull/561/files#r1265023182) discussion, I think a better name for this document would be "A request-response interface for protocols on any transport".

Reply:
renamed